src/Runtime.Base/src/System/Runtime/ExceptionHandling.cs

```diff
@@ -410,19 +398,20 @@ internal enum ExKind : byte
             None = 0,
             Throw = 1,
             HardwareFault = 2,
-            // unused: 3
             KindMask = 3,
 
             RethrowFlag = 4,
-            Rethrow = 5,        // RethrowFlag | Throw
-            RethrowFault = 6,   // RethrowFlag | HardwareFault
+
+            SuperscededFlag = 8,
```

Comment:
It is used on the unmanaged side for debug-only checks: https://github.com/dotnet/corert/search?utf8=%E2%9C%93&q=EK_SuperscededFlag

Will fix the spelling...